libs/config/src/lib/applications.config.spec.ts

```
@@ -0,0 +1,61 @@
+      process.env['PROTOCOL'] = 'https';
+      process.env['GATEWAY_HOST'] = 'gateway.example.com';
+      process.env['GATEWAY_PORT'] = '4444';
+
+      const config = gatewayConfig();
+      expect(config).toEqual({
+        protocol: 'https',
+        host: 'gateway.example.com',
+        port: '4444',
+      });
+
+      delete process.env['PROTOCOL'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)